app/assets/styles/bulletin.css

```diff
@@ -46,123 +41,111 @@ hr.image {
 	font-size: 0.8em;
 }
 
-.bupage-hero--title {
+.wrapper {
+	padding: 2em;
+	max-width: 87vw;
+	margin-left: auto;
+	margin-right: auto;
+	margin-top: 4em;
+}
+
+.wrapper__title {
 	color: #000099;
 	font-weight: 300;
 	font-size: 3rem;
 	margin-bottom: 0;
 }
 
-.bupage-hero--subtitle {
+.wrapper__subtitle {
 	color: #808080; 
 	font-weight: 300;
 	font-size: 2rem;
 	margin-top: 0;
 	margin-bottom: 1.2em;
 }
```

Comment:
Let's change `wrapper` to `article` then we can we use `article__title` and `article__subtitle`.